Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -14,6 +15,7 @@
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Data;
 using Windows.UI.Xaml.Media;
+using TextBlock = Windows.UI.Xaml.Controls.TextBlock;
```

Comment:
I don't think that's needed because of the "using Windows.UI.Xaml.Controls;" above.

Reply:
Yes you are right. I think this got added by ReSharper for some reason and I didn't think of checking sorry.